Solutions/Ubiquiti/Data Connectors/Connector_Ubiquiti_agent.json

```diff
@@ -65,10 +65,16 @@
             "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**UbiquitiAuditEvent**](https://aka.ms/sentinel-UbiquitiUnifi-parser) which is deployed with the Azure Sentinel Solution.",

```

Comment:
We've updated this parser note back to the original state due to it being a standard format followed for Data Connectors that are included with Parsers in Solutions.